.gitignore

```diff
@@ -28,3 +27,6 @@ replay_pid*
 
 # Ignore Gradle build output directory
 build
+
+# Ignore IntellijIDEA cache directory
+.idea
```

Comment:
You might want to remove the already added files as well

Reply:
fixed